sdk/identity/identity/src/msal/nodeFlows/msalOpenBrowser.ts

```diff
@@ -19,6 +20,14 @@ export interface MSALOpenBrowserOptions extends MsalNodeOptions {
   redirectUri: string;
 }
 
+/**
+ * A call to open(), but mockable
+ * @internal
+ */
+export const interactiveBrowserMockable = {
```

Comment:
Is there a way to stub the imported `open`  module directly in order to avoid adding product code to enable a test scenario? [maybe like this but I'm not too familiar with sinon](https://sinonjs.org/how-to/stub-dependency/)

Reply:
As far as I know, sinon can't stub default exports 😵 I could be wrong though.